src/popup/pages/Dashboard.vue

```diff
@@ -89,14 +97,20 @@ export default defineComponent({
   },
   setup() {
     const store = useStore();
+    const router = useRouter();
 
     const {
       activeAccount,
       activeAccountSimplexLink,
     } = useAccounts({ store });
+    const { checkIfOpenTransferSendModal } = useDeepLinkApi({ router });
 
     const { isNodeMainnet, isNodeTestnet } = useAeSdk({ store });
 
+    onMounted(() => {
+      checkIfOpenTransferSendModal();
+    });
```

Comment:
Shouldn't this happen only in the account details?

Reply:
Only on the dashboard

Reply:
As this function has its own test for which page should be allowed to open the modal I suggest to move this invocation to the `router.beforeEach` located in `src/popup/router/index.ts`.